charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -251,6 +259,7 @@ controller:
   logLevel: 2
   userAgentExtra: "helm"
   nodeSelector: {}
+  deployMetadataLabelerContainer: false
```

Comment:
Can we put this parameter as something like `enabled` in the `sidecars.metadataLabeler` section? (Similar to what we do for snapshotter but don't call it `forceEnable`)